lib/matplotlib/pyplot.py

```diff
@@ -1131,106 +1131,11 @@ def subplots(nrows=1, ncols=1, sharex=False, sharey=False, squeeze=True,
         # same as
         plt.subplots(2, 2, sharex=True, sharey=True)
     """
```

Comment:
Not sure how to make this work, given that the APIs are subtly different (one returns just the new axes, the other returns a figure, axes pair).  Open to suggestions...